packages/plugin-ext-vscode/src/node/plugin-vscode-file-handler.ts

```diff
@@ -28,7 +28,11 @@ export class PluginVsCodeFileHandler implements PluginDeployerFileHandler {
     }
 
     accept(resolvedPlugin: PluginDeployerEntry): boolean {
-        return resolvedPlugin.isFile() && resolvedPlugin.path() !== null && resolvedPlugin.path().endsWith('.vsix');
+        if (!resolvedPlugin.isFile()) {
+            return false;
+        }
+        const pluginPath = resolvedPlugin.path();
+        return !!pluginPath && pluginPath.endsWith('.vsix') || pluginPath.endsWith('gz');
```

Comment:
missing dot before gz ?

Reply:
i was not sure what is an extension, seems to be `.tgz` right now, but `.tar.gz` is another variant

Reply:
https://docs.npmjs.com/cli/pack.html

it seems it's always `.tgz` no ?

Reply:
fixed